tests/acceptance/pageObjects/FilesPageElement/publicLinksDialog.js

```diff
@@ -1,25 +1,244 @@
+const assert = require('assert')
+const util = require('util')
+
 module.exports = {
   commands: {
+    /**
+     * function clicks the edit button from public link dialog box
+     * @param linkName Name of the public link
+     * @returns {Promise<void>}
+     */
+    clickLinkEditBtn: async function (linkName) {
```

Comment:
```suggestion
    editLink: async function (linkName) {
```

Reply:
this function doesn't edit link you fat boy! :hocho:

Reply:
`clickLinkEditBtn` is not device-agnostic. What if the user `presses` instead of `clicking`?

P.S. Please read through [ownCloud Code of Conduct](https://owncloud.org/community/code-of-conduct/), especially, the `Be respectful` section.

Reply:
The function is made only to click the link edit button! IMO the name is not that confusing!